clang/test/Sema/lit.local.cfg

```diff
@@ -2,3 +2,15 @@ config.substitutions = list(config.substitutions)
 config.substitutions.insert(
     0, (r"%clang\b", """*** Do not use the driver in Sema tests. ***""")
 )
+
+if lit_config.update_tests:
+    import sys
+    import os
+
+    curdir = os.path.dirname(os.path.realpath(__file__))
+    testdir = os.path.dirname(curdir)
+    clangdir = os.path.dirname(testdir)
+    utilspath = os.path.join(clangdir, "utils")
```

Comment:
I believe this can be simplified to `os.path.join(clangdir, "..", "..", "utils")`?

Reply:
Ah neat, wasn't aware that was a thing